presto-geospatial/src/test/java/io/prestosql/plugin/geospatial/aggregation/TestGeometryUnionGeoAggregation.java

```diff
@@ -305,49 +305,49 @@
     public void testPoint(String testDescription, String expectedWkt, String... wkts)
```

Comment:
commit message is too long

Reply:
Thanks for your review, shorten the commit message already.